### DISCUSSION TITLE
Drawing / writing on slides (HTML)

### DISCUSSION THREAD

**Discussion by reparke** [marp-team/marp#66] [Q&A] [answered]
Are there any plans / is there a roadmap to add the ability to draw on marp-generated HTML slide presentations?

**Comment by yhatt** [comment, marked as answer]
No, there is no plans currently.

But we're welcome to submit request to [Marp CLI issue tracker](https://github.com/marp-team/marp-cli/issues)! (Marp CLI is generating HTML included many presenter tools)
